src/components/DatePickerRange.react.js

```diff
@@ -117,46 +117,63 @@ export default class DatePickerRange extends Component {
             stay_open_on_select,
             with_full_screen_portal,
             with_portal,
+            loading_state,
+            id,
         } = this.props;
 
         const verticalFlag = calendar_orientation !== 'vertical';
 
+        const DatePickerWrapperStyles = {
+            position: 'relative',
+            display: 'inline-block',
+        };
+
         return (
-            <DateRangePicker
-                daySize={day_size}
-                disabled={disabled}
-                displayFormat={display_format}
-                enableOutsideDays={show_outside_days}
-                endDate={end_date}
-                endDatePlaceholderText={end_date_placeholder_text}
-                firstDayOfWeek={first_day_of_week}
-                focusedInput={focusedInput}
-                initialVisibleMonth={() => {
-                    if (initial_visible_month) {
-                        return initial_visible_month;
-                    } else if (end_date && focusedInput === 'endDate') {
-                        return end_date;
-                    } else if (start_date && focusedInput === 'startDate') {
+            <div
```

Comment:
I have concerns over the impact these wrapper divs will have on the overall application display / flow.
These will default to `display: block` while the now-wrapped component may or may not have been that way.

What was the impact on the visual tests?

... Oh! Is see the default style above! As this been done for all comps?

Reply:
Just for these ones, because those were the only ones that showed visual changes.